dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -2049,13 +2053,22 @@ const hoverPagination = (format: ArticleFormat) => {
 	}
 };
 
+const decidePaletteDark = (format: ArticleFormat): DarkPaletteColours => {
+	return {
+		text: {
+			headline: textHeadlineDark(format),
+		},
+	};
+};
+
 export const decidePalette = (
 	format: ArticleFormat,
 	containerPalette?: DCRContainerPalette,
 ): Palette => {
 	const overrides =
 		containerPalette && decideContainerOverrides(containerPalette);
 	return {
+		dark: decidePaletteDark(format),
 		text: {
```

Comment:
It'd be a big change but I'd be very open to moving the existing properties into a palette.light, instead of having them at the top level, that way the palette structure is 

```ts
{
    light: { text: { ... } },
    dark: { text: { ... } }.
}
```

It'd be a lot of files to change so worth doing in another PR, but I think this would make it really clear that we support both light & dark mode when working on the project, and also lay the groundwork for expansion to support dark mode on the site in the future!